data/json/monster_factions.json

```
@@ -431,5 +431,19 @@
     "base_faction": "insect",
     "neutral": [ "bot", "insect", "animal", "zombie" ],
     "hate": [ "fish", "zombie_aquatic" ]
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "seweranha",
+    "base_faction": "mutant",
+    "hate": "fish",
+    "neutral": "seweranha"
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "sewer_snake",
+    "base_faction": "mutant",
+    "hate": "fish",
+    "neutral": "sewer_snake"
```

```suggestion
    "neutral": "seweranha"
  },
  {
    "type": "MONSTER_FACTION",
    "name": "sewer_snake",
    "base_faction": "mutant",
    "neutral": "sewer_snake"
```

Alternatively, fish need to be something other than neutral toward these two; I can push a PR to your branch to do that and some cleanup (grouping all mutant piscivores into a base faction with subfactions, so we can just then tell fish to not be neutral toward mutant_piscivores). Which do you prefer? @Venera3, any comments?

---

If you wish fish not be neutral towards these two new factions, then how? Friendly? Hate? By mood?

---

Probably the same way they are toward the other two mutant factions that eat them (gator and crayfish) - hate.

---

Whoops, crayfish are insect faction, not mutant - leaving out of new "mutant_piscivores" proposed faction (working on PR to your branch).

---

Ok, applied, thanks.